### PR TITLE
Adding README for forcedroid and forceios

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: android/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+In this case, you can access the forcedroid app at `[Install Directory]/node_modules/.bin/forcedroid`.
+
+## Using forcedroid
+
+For the rest of this document, we'll assume that `forcedroid` is on your path.
+
+Typing `forcedroid` with no arguments gives you a breakdown of the usage:
+
+        $ forcedroid
```
Comment (bhariharan) [inline android/README.md]: I should probably update these arguments to match the latest.

**Comment by wmathurin** [comment]
@rwhitley should we still have all those links to the wiki?

File: android/README.md (hunk 1 above)
Comment (wmathurin) [inline android/README.md]: You can go ahead and merge, and I can send a PR with the changes to the README.md.

Comment (bhariharan) [inline android/README.md]: @wmathurin I made the changes in my latest commit.

File: android/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+**App Name:** The name of your application
+
+**Target App Folder:** The folder where you want your app to be created.  Your app will be contained in a folder underneath this folder, alongside a `forcedroid` folder containing the Mobile SDK libraries that your app is linked to.
+
+**App Package Identifier:** The Java package identifier for your app (e.g. `com.acme.mobile_apps`).  **Note:** Your package name must be formatted as a [valid Java package name](http://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html), or you will receive an error.
+
+**Start Page:** \( *Required for hybrid\_remote apps only* \) The starting page of your application on salesforce.com.  This is the entry point of your remote application, though it's only the path, not the server portion of the URL.  For instance, `/apex/MyVisualforceStartPage`.
+
+**Use SmartStore:** \( *optional* \) Whether to use SmartStore in your app.  The value is `false` by default.  Set this value to `true` if you intend to use SmartStore in your app.
```
Comment (wmathurin) [inline android/README.md]: This one is gone

File: android/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+**App Type:** The type of application you wish to develop:
+
+- **native** — A fully native Android application
+- **hybrid\_remote** — A hybrid application, based on the [Cordova](http://cordova.apache.org/) framework, that runs in a native container.  The app contents live in the cloud as a [Visualforce](http://wiki.developerforce.com/page/An_Introduction_to_Visualforce) application
+- **hybrid\_local** — A hybrid application, based on the Cordova framework, that runs in a native container.  The app contents are developed locally in the Eclipse project, and are deployed to the device itself when the app is built
+
+**App Name:** The name of your application
+
+**Target App Folder:** The folder where you want your app to be created.  Your app will be contained in a folder underneath this folder, alongside a `forcedroid` folder containing the Mobile SDK libraries that your app is linked to.
```
Comment (wmathurin) [inline android/README.md]: There is no forcedroid folder.

File: ios/README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+**App Type:** The type of application you wish to develop:
+
+- **native** — A fully native iOS application
+- **hybrid\_remote** — A hybrid application, based on the [Cordova](http://cordova.apache.org/) framework, that runs in a native container.  The app contents live in the cloud as a [Visualforce](http://wiki.developerforce.com/page/An_Introduction_to_Visualforce) application
+- **hybrid\_local** — A hybrid application, based on the Cordova framework, that runs in a native container.  The app contents are developed locally in the Xcode project, and are deployed to the device itself when the app is built
+
+**App Name:** The name of your application
+
+**Company ID:** An identifier for your company, similar to a Java package (e.g. `com.acme.MobileApps`).  This concatenates with the app name to form the unique identifier for your app in the App Store.
```
Comment (wmathurin) [inline ios/README.md]: We called it package name for iOS also

File: ios/README.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+**Organization:** The name of your company or organization.  For example, `Acme Widgets, Inc.`
+
+**Start Page:** \( *Required for hybrid\_remote apps only* \) The starting page of your application on salesforce.com.  This is the entry point of your remote application, though it's only the path, not the server portion of the URL.  For instance, `/apex/MyVisualforceStartPage`.
+
+**Output Directory:** \( *optional* \) The directory where you want your app to be created.  If not specified, it will be created in your current working directory.
```
Comment (wmathurin) [inline ios/README.md]: We should copy those two lines into the forcedroid readme - because it behaves that way now

File: ios/README.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+**App ID:** \( *optional* \) The Connected App Consumer Key that identifies your app in the cloud.  This argument defaults to a sample key to allow you to test your app.  However, **you _must_ specify your own Consumer Key before you submit your app to the App Store**.
```
Comment (wmathurin) [inline ios/README.md]: Gone

File: ios/README.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+**Callback URI:** \( *optional* \) The Callback URL associated with your Connected App.  As with the App ID, this argument defaults to a value for a sample app.  **You _must_ specify your own Callback URL before you submit your app to the App Store**.
```
Comment (wmathurin) [inline ios/README.md]: Gone

File: ios/README.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+## More information
+
+- The Salesforce Mobile SDK for Android (and package) source repository lives [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android).
+
+- See [our developerforce site](http://wiki.developerforce.com/page/Mobile_SDK) for more information about how you can leverage the Salesforce Mobile SDK with the force.com platform.
```
Comment (wmathurin) [inline ios/README.md]: @rwhitley Is the wiki retired?

Comment (bhariharan) [inline ios/README.md]: The link still works. I spoke with Richard, he said we should keep the link till it moves to the new domain (that work hasn't started yet).

**Pull request merged**